xbmc/interfaces/legacy/Control.h

```diff
@@ -1068,22 +1068,24 @@ namespace XBMCAddon
       /// Sets the type of this edit control.
       ///
       /// @param type              integer - type of the edit control.
-      /// | Param                            | Definition                                  |
-      /// |----------------------------------|:--------------------------------------------|
-      /// | xbmcgui.INPUT_TYPE_TEXT          | (standard keyboard)
-      /// | xbmcgui.INPUT_TYPE_NUMBER        | (format: #)
-      /// | xbmcgui.INPUT_TYPE_DATE          | (format: DD/MM/YYYY)
-      /// | xbmcgui.INPUT_TYPE_TIME          | (format: HH:MM)
-      /// | xbmcgui.INPUT_TYPE_IPADDRESS     | (format: #.#.#.#)
-      /// | xbmcgui.INPUT_TYPE_PASSWORD      | (input is masked)
-      /// | xbmcgui.INPUT_TYPE_PASSWORD_MD5  | (input is masked, return md5 hash of input)
-      /// | xbmcgui.INPUT_TYPE_SECONDS       | (format: SS or MM:SS or HH:MM:SS or MM min)
+      /// | Param                                         | Definition                                  |
+      /// |-----------------------------------------------|:--------------------------------------------|
+      /// | xbmcgui.INPUT_TYPE_TEXT                       | (standard keyboard)
+      /// | xbmcgui.INPUT_TYPE_NUMBER                     | (format: #)
+      /// | xbmcgui.INPUT_TYPE_DATE                       | (format: DD/MM/YYYY)
+      /// | xbmcgui.INPUT_TYPE_TIME                       | (format: HH:MM)
+      /// | xbmcgui.INPUT_TYPE_IPADDRESS                  | (format: #.#.#.#)
+      /// | xbmcgui.INPUT_TYPE_PASSWORD                   | (input is masked)
+      /// | xbmcgui.INPUT_TYPE_PASSWORD_MD5               | (input is masked, return md5 hash of input)
+      /// | xbmcgui.INPUT_TYPE_SECONDS                    | (format: SS or MM:SS or HH:MM:SS or MM min)
+      /// | xbmcgui.INPUT_TYPE_PASSWORD_NUMBER_VERIFY_NEW | (numeric input is masked)
```

Comment:
Can you explain why the "verify new" in the constant name? Why not simply INPUT_TYPE_NUMBER_MASKED?

Reply:
these are 1:1 mappings to:
https://github.com/xbmc/xbmc/blob/d3a641fe5132cb6b2ea016e2a2c1d8eaf794795e/xbmc/guilib/GUIEditControl.h#L26-L42

guess the 'verify' bit relates to the fact you need to enter a password twice when setting it,
the second time is done to verify you didn't make a typo.

Reply:
I guess what I really need, which I suspect is going to be more work, is a new `INPUT_TYPE_PASSWORD_NUMBER` that would accept only masked digits and that would return the entered digits. Ideally with a configurable heading. :smile:

Reply:
Maybe remove then direct on description text the `(...)`, think is from time where it was not included inside a table:
```
xbmcgui.INPUT_TYPE_TEXT (standard keyboard)
```